Bio/Entrez/Parser.py

```diff
@@ -328,7 +328,7 @@ def read(self, handle):
         if hasattr(handle, "closed") and handle.closed:
             # Should avoid a possible Segmentation Fault, see:
             # http://bugs.python.org/issue4877
-            raise IOError("Can't parse a closed handle")
+            raise OSError("Can't parse a closed handle")
```

Comment:
I'm not sure if this should be changed here, for the record, but would it be an idea to change this to `ValueError` instead?

Thea reason is `read()` on a closed file handle raises a `ValueError` on Python 3 (actually on Python 2 as well):

```
Python 3.8.1 (default, Dec 21 2019, 20:57:38) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> f = open(<fname>, "r")
>>> f.close()
>>> f.read()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: I/O operation on closed file.
```
```

Reply:
We don't need to check for closed files at all. See the original issue in http://bugs.python.org/issue4877 ; this was fixed in 2011.

Reply:
So that if-clause can be removed entirely?